contracts/src/ERC20Source.sol

```diff
@@ -38,10 +38,13 @@ contract ERC20Source is IERC20Source, TeleporterTokenSource {
     IERC20 public immutable token;
 
     /**
-     * @notice Initializes this source token bridge instance to send
-     * tokens to the specified destination chain and token bridge instance.
-     *
-     * Teleporter fees are paid by the same token that is being bridged.
+     * @notice Initializes this source token bridge instance to send ERC20
+     * tokens to other destination token bridges.
+     * @param teleporterRegistryAddress The current blockchain id's Teleporter registry
+     * address. https://github.com/ava-labs/teleporter/tree/main/contracts/src/Teleporter/upgrades
```

Comment:
```suggestion
     * address. See here for details: https://github.com/ava-labs/teleporter/tree/main/contracts/src/Teleporter/upgrades
```

Reply:
Another very minor nit: technically a chain could have multiple registry contracts. Could we reword to "The address of the Teleporter registry to be used by this contract."?

Or if you feel strongly about the wording as is just capitalize "ID" :)

Reply:
I prefer to leave current wording, because even though it can have multiple registry contracts, think it's better to push for just one as a standard. Capitalized the ID.